QOwnNotes-Project.pro

```diff
@@ -0,0 +1,13 @@
+TEMPLATE = subdirs
+
+CONFIG += c++11
+
+SUBDIRS = \
+    src \
+    tests
+
+src.file = src/QOwnNotes.pro
+tests.file = tests/QOwnNotesTests.pro
+
+test.depends = src
+ 
```

Comment:
Why the name `QOwnNotes-Project.pro`? Wouldn't `QOwnNotes.pro` be better?

Reply:
Just to differentiate from the inner QOwnNotes.pro file :grimacing:

Reply:
Renamed inner files to `src.pro` and `tests.pro`. Outer file is now `QOwnNotes.pro`

Reply:
But I still don't like that `QOwnNotes-Project.pro`. Why do we need that again?

Reply:
Just for utility. Tests and Project are opened as one overall project with two subprojects(app, tests) so you can quickly switch from one to another

Reply:
Isn't that almost more confusing than before? Are there other projects who are using one project file for the application and the tests?

Reply:
Yeah most of them

But I will remove it.

Reply:
I'm not against it if it is considered "best practice". :laughing: And that was what I tried to find out.

Reply:
I don't know if its best practice lol :rofl: , but it does kinda make it easier to run the tests(from Qt creator, i.e., just switch the target)

Reply:
I just switch project.